static/js/tools/scatter2/info.tsx

```diff
@@ -37,9 +37,96 @@ function Info(): JSX.Element {
           to choose from, arranged in a topical hierarchy.
         </li>
       </ol>
-
-      {/* TODO(intrepiditee): Add descriptions examples */}
-
+      <p>Or you can start your exploration from these interesting points ...</p>
```

Comment:
We should run these examples with the teams too after it's pushed to auotpush.

Reply:
yeup, sounds good. It's also already on dev